pkg/mountutil/mountutil.go

```diff
@@ -90,7 +96,7 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		return nil, errors.Errorf("failed to parse %q", s)
 	}
 	res.Mount = specs.Mount{
-		Type:        "none",
+		Type:        string(vType),
```

Comment:
This is an OCI Mount object, you can’t mix up Docker API type here

Reply:
@AkihiroSuda does OCI provided types ? or I may set the directly in the code ?

Reply:
@AkihiroSuda  is it ok for you ?